src/panels/lovelace/ha-panel-lovelace.ts

```diff
@@ -199,33 +208,67 @@ export class LovelacePanel extends LitElement {
         oldHass.floors !== this.hass.floors
       ) {
         if (this.hass.config.state === "RUNNING") {
-          this._askRefreshConfig();
+          this._debounceRegistriesChanged();
         }
       }
       // If ha started, refresh the config
       if (
         this.hass.config.state === "RUNNING" &&
         oldHass.config.state !== "RUNNING"
       ) {
-        this._refreshConfig();
+        this._regenerateStrategyConfig();
```

Comment:
This should already be handled by the loading card in the strategy right?

Reply:
All strategies doesn't implement the loading card but I think it's fine to also remove this code.

Reply:
Yeah, I think it is best if we remove it from the loading card and handle it here for all